zubhub_frontend/zubhub/src/views/create_activity/CreateEditActivity.jsx

```diff
@@ -229,18 +242,30 @@ export default function CreateActivity(props) {
               Previous
             </CustomButton>
           )}
-
+          {activeStep === 2 && (
```

Comment:
Thought it is ideally nice to have drafting and publishing abilities differently. This button sets the activity as a draft by just making `publish: false`. 
Still open for suggestions @tuxology